wp-content/plugins/core/src/Blocks/Types/Gallery_Slider/Gallery_Slider.php

```diff
@@ -51,30 +51,30 @@ public function add_fields(): void {
 		//==========================================
 		// Content Fields
 		//==========================================
-		$this->add_section( new Field_Section( self::SECTION_CONTENT, __( 'Content', 'tribe' ), 'accordion' ) )
+		$this->add_section( new Field_Section( self::SECTION_CONTENT, esc_html__( 'Content', 'tribe' ), 'accordion' ) )
 			->add_field( new Field( self::NAME . '_' . self::TITLE, [
-					'label' => __( 'Title', 'tribe' ),
+					'label' => esc_html__( 'Title', 'tribe' ),
 					'name'  => self::TITLE,
 					'type'  => 'text',
 				] )
 			)->add_field( new Field( self::NAME . '_' . self::DESCRIPTION, [
-					'label'        => __( 'Description', 'tribe' ),
+					'label'        => esc_html__( 'Description', 'tribe' ),
 					'name'         => self::DESCRIPTION,
 					'type'         => 'wysiwyg',
 					'toolbar'      => 'basic',
 					'media_upload' => 0,
 				] )
 			)->add_field( new Field( self::NAME . '_' . self::CTA, [
-					'label' => __( 'Call to Action', 'tribe' ),
+					'label' => esc_html__( 'Call to Action', 'tribe' ),
 					'name'  => self::CTA,
 					'type'  => 'link',
 				] )
 			)->add_field( new Field( self::NAME . '_' . self::GALLERY, [
-					'label'        => __( 'Gallery', 'tribe' ),
+					'label'        => esc_html__( 'Gallery', 'tribe' ),
 					'name'         => self::GALLERY,
 					'type'         => 'gallery',
 					'max'          => 12,
-					'instructions' => __(
+					'instructions' => esc_html__(
```

Comment:
🔴 
The text here includes HTML. In this case the <br> will be output as `&lt;br&gt;`. We might need to break this down into multiple esc_html__() functions to allow for new lines in the correct spaces.

Reply:
Thank you for noticing 👍